### ISSUE TITLE
About: Revise links on "What Is Quire" to include action button + image

### ISSUE THREAD

**Issue by Erin-Cecele** [gettypubs/quire#262]
If this works, update elsewhere on the website.

**Comment by Erin-Cecele** [comment]
@geealbers What is the best way to center the action buttons on the page? I played around with the CSS and looked at the homepage (as those are centered) but couldn't find solution.

<img width="679" alt="Screen Shot 2020-09-08 at 5 36 22 PM" src="https://user-images.githubusercontent.com/62075223/92541040-d3283600-f1f9-11ea-8f4d-d1fab21b830e.png">

**Comment by geealbers** [comment]
In custom.css around line 70 I think, try changing from:

```css
.quire-page__content .container .content .action-button {
  display: inline-block;
}
```

to:

```css
.quire-page__content .container .content .action-button {
  text-align: center;
}
```

**Comment by Erin-Cecele** [comment]
Great! I created a new button: "action-button-center" and added to What is Quire, History, and Forum. On the pages where the action buttons are centered. I also bolded the text. I like the way this looks but don't know if there's an issue with inconsistency with the other pages's buttons. 

Either way, I think our problem of skimming past these links has hopefully been solved. @evale124 and @geealbers Once merged, let me know what you think of this change!
